xla/service/gpu/BUILD

```diff
@@ -1343,12 +1339,7 @@ cc_library(
         "//xla/stream_executor",
         "@tsl//tsl/platform:logging",
         "@tsl//tsl/platform:statusor",
-    ]) + if_cuda_is_configured([
```

Comment:
Why are these dependencies not needed anymore?

Reply:
yes, this is because gpublas_lt_matmul_thunk does not use cublas_lt/hiblas_lt directly. Instead, it depends on matmul_utils library which contains all blas-lt related deps. 

The same holds, for example, for gemm_algorithm_picker which also uses blas-lt API through mamtul_utils:
https://github.com/openxla/xla/blob/main/xla/service/gpu/BUILD#L1334-L1346

Reply:
yes, this is because gpublas_lt_matmul_thunk does not use cublas_lt/hiblas_lt directly. Instead, it depends on matmul_utils library which contains all blas-lt related deps.

The same holds, for example, for gemm_rewriter which also uses blas-lt API through mamtul_utils:
https://github.com/openxla/xla/blob/main/xla/service/gpu/BUILD#L1116-L1147

Reply:
Thanks for the explanation.